C/common/config_category.cpp

```diff
@@ -905,6 +905,31 @@ ConfigCategory::CategoryItem::CategoryItem(const string& name, const std::string
 	m_itemType = StringItem;
 }
 
+/**
+ * Copy constructor for configuration item
+ */
+ConfigCategory::CategoryItem::CategoryItem(const CategoryItem& rhs)
+{
+	m_name = rhs.m_name;
+	m_displayName = rhs.m_displayName;
+        m_type = rhs.m_type;
+        m_default = rhs.m_default;
+        m_value = rhs.m_value;
+        m_description = rhs.m_description;
+        m_order = rhs.m_order;
+        m_readonly = rhs.m_readonly;
+        m_deprecated = rhs.m_deprecated;
+        m_minimum = rhs.m_minimum;
+        m_maximum = rhs.m_maximum;
+        m_filename = rhs.m_filename;
```

Comment:
Wrong indentation

Reply:
No, it's an issue with git and tabs - view it in an editor and it is fine